clustersmgmt/v1/aws_infrastructure_access_roles_client.go

```diff
@@ -103,7 +103,7 @@ func (r *AWSInfrastructureAccessRolesListRequest) Header(name string, value inte
 // The syntax of this parameter is similar to the syntax of the _order by_ clause of
 // a SQL statement, but using the names of the attributes of the role instead of
 // the names of the columns of a table. For example, in order to sort the roles
-// descending by dislay_name the value should be:
```

Comment:
This is generated code. If you change it here the tests will fail because they detect a change in generated code. This typo should be fixed in the model.

The other changes are OK.

Reply:
Fixed